### PR TITLE
PR and Issue Templates

### PR DESCRIPTION
**Pull request by niteshkumarniranjan**
Added PR and Issue Templates
Solves: #31

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.
+
+- [ ] Included a Preview link and screenshot showning after and before the changes.
+- [ ] Included a description of change below.
+- [ ] Squashed the commits.
+
+# Changes done in this Pull Request
+
+- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
+- Fixes`#<Issue no.>`
```
Comment (fragm3) [inline .github/PULL_REQUEST_TEMPLATE.md]: Add Preview link, Description/Changes just like Fixes.

**Comment by fragm3** [comment]
@niteshkumarniranjan make changes for ISSUE_TEMPLATE file also, see above review

**Comment by niteshkumarniranjan** [comment]
@fragm3 Done

**Comment by sarnava1** [comment]
LGTM!

**Pull request merged**